content/docs/start/data-pipelines.md

```diff
@@ -217,30 +219,29 @@ Please check the `dvc.yaml` again, it should have one more stage now.
 
 </details>
 
-This should be a good point to commit the changes with Git. These include
+This should be a good time to commit the changes with Git. These include
 `.gitignore`, `dvc.lock`, and `dvc.yaml` — which describe our pipeline.
 
 ## Reproduce
 
-The whole point of creating this `dvc.yaml` pipelines file is an ability to
-reproduce the pipeline:
+The whole point of creating this `dvc.yaml` file is the ability to easily
+reproduce a pipeline:
 
 ```dvc
 $ dvc repro
 ```
 
 <details>
 
-### ⚙️ Expand to have some fun with it
+### ⚙️ Expand to have some fun with it 💃
 
 Let's try to play a little bit with it. First, let's try to change one of the
 parameters for the training stage:
 
-```dvc
-$ vim params.yaml
-```
+- Open `params.yaml` and change `n_est` to `100`, and
+- (re)run `dvc repro`.
```

Comment:
I'm not sure splitting a sentence into 2 bullets is readable. Maybe just make it a normal sentence as part of the previous paragraph? Along with "you should see" below, as well

> We actually have some guidelines on bullet lists BTW (please see https://dvc.org/doc/user-guide/contributing/docs if you haven't).

Reply:
I think it makes it much more readable in this case, clearly splitting into 2 steps without the odd insistence on `vim`.

btw I just read https://dvc.org/doc/user-guide/contributing/docs (only read up to dev env before)... it's comically self-conflicted and also occasionally manages to be wrong from both US & UK perspectives. May have to fix it at some point.

I can find about 5 things wrong with the docs bullet point on, well, bullet points; but don't see anything adverse to this change.

Reply:
Yes, def. better than `vim`. Just not sure about the syntax (a sentence / bullet list cross-breed).

> splitting into 2 steps

Maybe just change bullets to numbers?

Reply:
> I can find about 5 things wrong with the docs bullet point
it's comically self-conflicted and also occasionally manages to be wrong from both US & UK perspectives. May have to fix it

Yes, please propose something.

Reply:
> Maybe just change bullets to numbers?

done